angularSTEP/src/app/app.module.ts

```diff
@@ -94,18 +99,23 @@ import {ViewProfilesComponent} from './view-profiles/view-profiles.component';
     MatDividerModule,
     MatFormFieldModule,
     MatButtonModule,
+    MatExpansionModule,
     MatFormFieldModule,
     MatInputModule,
     MatIconModule,
     MatFormFieldModule,
     MatAutocompleteModule,
     ReactiveFormsModule,
+    MatListModule,
     MatMenuModule,
     MatProgressBarModule,
     MatRippleModule,
     MatProgressBarModule,
     MatProgressSpinnerModule,
+    MatRadioModule,
     MatSelectModule,
+    MatSidenavModule,
+    MatSlideToggleModule,
     MatStepperModule,
     MatToolbarModule,
     ReactiveFormsModule,
```

Comment:
Duplicate `ReactiveFormsModule` 
Not too big a deal but this just got corrected in main so we should try and keep the duplication from occurring again

Reply:
Wow good eye! Thank you